### DISCUSSION TITLE
QuickGrid Blazor .NET 8 is acting up. How to fix?

### DISCUSSION THREAD

**Discussion by NikolayRenziglov** [dotnet/aspnetcore#55001] [Q&A]
Hi there.
After a while I have switched to Blazor .NET 8 I noticed that the QuickGrid Blazor stopped working. The symptom is that it is flickering/jittering while scrolling and stay in that mode indefinitely. When tracing, I can see Skip grows from 0 while I'm gradually scrolling grid down and suddenly it gets back to Skip=0, Take=say 30. That marks it is broken, it gets jittering even without touching a scroll bar. What can be a reason? 
Thanks

**Comment by NikolayRenziglov** [comment]
Additionally: 1. it worked Ok on .NET 6 for sure 2. It uses virtualization 3. It uses ItemsProvider for remote data access. Thanks

**Comment by micahosborne** [comment]
Having the same issue, things are jumping all over the place as i scroll, nothing is word wrapping either.
